pyanaconda/modules/common/errors/installation.py

```diff
@@ -0,0 +1,33 @@
+#
+# DBus errors related to tasks and the installation.
```

Comment:
`installation tasks and the installation` sounds better to me.

Reply:
Task errors moved to task.py.